contrib/doc-gen/generate.go

```diff
@@ -0,0 +1,152 @@
+// This is a golang script, needed for generating the RPC bitcoin documentation
+//
+// What is necessary to run this:
+// (1) install golang
+// (2) install bitcoin core, set it up to use regtest
+// (3) run bitcoind
+// (4) change BITCOIN_PATH to actual bitcoin-cli directory path, change VERSION
+// (5) run this script with `go run generate.go` while being in contrib/doc-gen
+// (6) add the generated files to git
+package main
+
+import (
+	"fmt"
+	"text/template"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const BITCOIN_PATH = "/home/g/dev/bitcoind/bitcoin-0.16.0/bin/bitcoin-cli"
```

Comment:
This should be changed. Since `bitcoin-cli` versions is generally forwards- and backwards-compatible, using whichever version is in `$PATH` should be fine.

Reply:
Yes